### PR TITLE
feat(console,phrases): integrate jwt customizer api

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Integrate JWT customizer API.

This PR includes the following updates:
1. Extract a Main component. Render the form elements after the fetch request is completed
2. Rename the frontend `jwt-claims` pathName with backend API naming convention `jwt-customizer`
3. Integrate the GET `api/configs/jwt-customizer/:type` fetch API
4. Integrate the PUT `api/config/jwt-customizer/:type` form submit api
5. Add a front-end test sample code validator. The input code must be a valid JSON. 

## Testing

test locally. Integration tests will be added later

## Checklist

- [ ] ~`.changeset`~
- [ ] ~unit tests~
- [ ] ~integration tests~
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/pages/JwtClaims/SettingsSection/EnvironmentVariablesField.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ function EnvironmentVariablesField() {
 
   const valueValidator = useCallback(
     (value: string, index: number) => {
-      return getValues(`environmentVariables.${index}.value`)
+      return getValues(`environmentVariables.${index}.key`)
```
Comment (simeng-li) [inline packages/console/src/pages/JwtClaims/SettingsSection/EnvironmentVariablesField.tsx:66]: fix a bug. if the key is not empty, the value must not be empty.

File: packages/console/src/pages/JwtClaims/utils.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+const formatSampleCodeStringToJson = (sampleCode?: string) => {
+  if (!sampleCode) {
+    return;
+  }
+
+  try {
+    // eslint-disable-next-line no-restricted-syntax -- guarded by back-end validation
+    return JSON.parse(sampleCode) as Record<string, unknown>;
+  } catch {}
```
Comment (charIeszhao) [inline packages/console/src/pages/JwtClaims/utils.ts:71]: nit: Should we use a zod guard and safeParse the json?

**Review by charIeszhao — APPROVED** [review]
Overall looks good!

File: packages/console/src/pages/JwtClaims/utils.ts (hunk 2 above)
Comment (simeng-li) [reply packages/console/src/pages/JwtClaims/utils.ts:71]: Have an improvement issue tracking this. As mentioned in the comment, it relies on the API side input guard.  Need to have a comprehensive design to display the front-end guard error and data structure info.

**Pull request merged**